pkg/types/ccipocr3/generic_types.go

```diff
@@ -204,8 +204,8 @@ func (m Message) String() string {
 
 // IsEmpty returns true if the message is empty. Can't use == Message{} without using reflect.DeepEqual.
 func (m Message) IsEmpty() bool {
```

Comment:
Can we check where this is used so we don't accidentally introduce a bug due to change of semantics?

Reply:
Actually I'd rather have a new method called `IsPseudoDeleted` because `IsEmpty` currently doesn't actually check emptiness.